logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/CallManager.kt

```diff
@@ -24,5 +24,3 @@ interface CallManager {
     suspend fun updateVideoState(conversationId: ConversationId, videoState: VideoState)
     suspend fun requestVideoStreams(conversationId: ConversationId, callClients: CallClientList)
 }
-
-expect class CallManagerImpl : CallManager
```

Comment:
Do we need to remove it ? We used the `expect` modifier here to make the compiler ensures that every function has its implementation in each platform.

Reply:
No, we can keep it but it also doesn't really serve any purpose as long as we have an `expect GlobalCallManager` because it instantiates the platform specific version of the `CallManager`.